flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/AbstractBlockingResultInfo.java

```diff
@@ -72,4 +73,15 @@ public void resetPartitionInfo(int partitionIndex) {
     int getNumOfRecordedPartitions() {
         return subpartitionBytesByPartitionIndex.size();
     }
+
+    /**
+     * Aggregate the input number bytes of a subtask according to the index range for partition and
+     * subpartition.
+     *
+     * @param partitionIndexRange Range of the index of the consumed partition.
+     * @param subpartitionIndexRange Range of the index of the consumed subpartition.
+     * @return Aggregated input number bytes of current result info.
+     */
+    public abstract long getExecutionVertexInputNumBytes(
```

Comment:
Mayebe `getExecutionVertexInputNumBytes` -> `getNumBytes`, because the execution vertex input should be transparent to the blocking result info. And we should add it to the interface `BlockingResultInfo`

Reply:
Fix this.